hail/python/hail/backend/backend.py

```diff
@@ -151,7 +151,7 @@ def get_reference(self, name):
         return json.loads(Env.hail().variant.ReferenceGenome.getReference(name).toJSONString())
 
     def add_sequence(self, name, fasta_file, index_file):
-        Env.hail().variant.ReferenceGenome.addSequence(name, fasta_file, index_file)
+        scala_object(Env.hail().variant, 'ReferenceGenome').addSequence(name, fasta_file, index_file)
```

Comment:
Why didn't this affect `remove_sequence` etc?

Reply:
same issue!

Reply:
What about `add_liftover` below? I'm confused why some of this seems to work but not others.